<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>业务办理</string>
  </property>
  <widget class="QLabel" name="banner">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/new/prefix1/banner3.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="bankL">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>银行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="accountIDL">
        <property name="text">
         <string>登陆账户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ownerIDL">
        <property name="text">
         <string>持卡人身份证号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="moneyL">
        <property name="text">
         <string>余额</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="addMoneyB">
          <property name="text">
           <string>存款</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="changePwB">
          <property name="text">
           <string>修改密码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="logoutB">
          <property name="text">
           <string>取卡登出</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="takeMoneyB">
          <property name="text">
           <string>取款</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
